book/src/prevent-volume-mode-conversion.md

```diff
@@ -0,0 +1,31 @@
+# Prevent unauthorised volume mode conversion
+
+## Status
+
+Status | Min K8s Version | Max K8s Version | external-snapshotter Version | external-provisioner Version
+--|--|--|--|--
+Alpha | 1.24 | - | 3.2+ | 6.0.1+
```

Comment:
The external-snapshotter and external-provisioner versions are reversed.

Reply:
fixed